packages/ra-language-english/src/index.ts

```diff
@@ -38,7 +38,7 @@ const englishMessages: TranslationMessages = {
             move_up: 'Move up',
             move_down: 'Move down',
             open: 'Open',
-            toggle_theme: 'Toggle Theme',
+            toggle_theme: 'Toggle light/dark mode',
```

Comment:
Did you check the french translations?

Reply:
Yes, the French translation is already "Thème clair/sombre", which is explicit. 

I changed the English translation because it kind of conflicted with the "change theme" tooltip of the ThemeSwapper button in the demo. Now it is more specific.